packages/common/src/compiler/filtersCompiler.ts

```diff
@@ -29,10 +29,12 @@ export const renderStringFilterSql = (
     const filterType = filter.operator;
     const escapedFilterValues = filter.values?.map((v) =>
         typeof v === 'string'
-            ? v.replaceAll(
-                  stringQuoteChar,
-                  `${escapeStringQuoteChar}${stringQuoteChar}`,
-              )
+            ? v
+                  .replaceAll(
+                      stringQuoteChar,
+                      `${escapeStringQuoteChar}${stringQuoteChar}`,
+                  )
+                  .replaceAll('', '\\_')
```

Comment:
```suggestion
                  .replaceAll('_', '\\_')
```

Would be good to also escape the rest of the wildcards. 
The `escapedFilterValues` is used by all filters, not just `LIKE` filters. Would they still work as intended ? 
eg: `name equals jo_se`

Reply:
How do I know that a character is a wildcard?
And yes, it breaks them 😭  could I isolate this to `not_include` filter?